frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card.tsx

```diff
@@ -55,8 +55,10 @@ const DataReductionCard: React.FC<DashboardItemProps> = ({
     'loadError',
   ]);
 
-  const efficiency = getGaugeValue(efficiencyQueryResult);
-  const savings = getGaugeValue(savingsQueryResult);
+  const efficiencyInfo = Number(getGaugeValue(efficiencyQueryResult));
+  const efficiency = efficiencyInfo === 0 ? 1 : efficiencyInfo;
+  const savingsInfo = Number(getGaugeValue(savingsQueryResult));
+  const savings = savingsInfo <= 0 ? 'No Savings' : String(savingsInfo);
```

Comment:
We can directly pass the prometheus results and then calculate the result and show message in ItemsBody.
This approach I am expecting.